spring-rabbit/src/main/java/org/springframework/amqp/rabbit/support/PublisherCallbackChannel.java

```diff
@@ -74,6 +83,14 @@
 	 */
 	Channel getDelegate();
 
+	/**
+	 * Set a callback to be invoked after the ack/nack has been handled.
+	 * @param callback the callback.
+	 * @param proxy the proxy.
+	 * @since 2.1
+	 */
+	void setAfterAckCallback(Consumer<ChannelProxy> callback, ChannelProxy proxy);
```

Comment:
Let's hope we don't have any package tangles with this !

Reply:
Good point - I'll check 😄